src/routes/search-results.js

```diff
@@ -5,30 +5,83 @@ import {
   Topbar
 } from '../components';
 
-// TODO: Delete this. This is only a temporary solution to the handle click
-//   callback.
 function handleClick() {
-  console.log("I've been clicked! Bless this day.");
+  console.log("Search Results card clicked.");
 }
```

Comment:
purpose?

Reply:
Left in for potential logging, but I can remove it!

Reply:
removed